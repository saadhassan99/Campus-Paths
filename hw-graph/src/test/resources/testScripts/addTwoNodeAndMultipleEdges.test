# Tests adding two nodes, multiple edges to graph.

# Create a graph
CreateGraph two_nodes_multi_edges_graph

# Add two nodes
AddNode two_nodes_multi_edges_graph a
AddNode two_nodes_multi_edges_graph b

# Add five edges
AddEdge two_nodes_multi_edges_graph a a AA
AddEdge two_nodes_multi_edges_graph a b AB1
AddEdge two_nodes_multi_edges_graph a b AB2
AddEdge two_nodes_multi_edges_graph b a BA
AddEdge two_nodes_multi_edges_graph b b BB

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_multi_edges_graph
ListChildren two_nodes_multi_edges_graph a
ListChildren two_nodes_multi_edges_graph b