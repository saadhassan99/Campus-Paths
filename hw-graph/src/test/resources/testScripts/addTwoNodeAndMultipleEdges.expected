# Tests adding two nodes, multiple edges to graph.

# Create a graph
created graph two_nodes_multi_edges_graph

# Add two nodes
added node a to two_nodes_multi_edges_graph
added node b to two_nodes_multi_edges_graph

# Add five edges
added edge AA from a to a in two_nodes_multi_edges_graph
added edge AB1 from a to b in two_nodes_multi_edges_graph
added edge AB2 from a to b in two_nodes_multi_edges_graph
added edge BA from b to a in two_nodes_multi_edges_graph
added edge BB from b to b in two_nodes_multi_edges_graph

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_multi_edges_graph contains: a b
the children of a in two_nodes_multi_edges_graph are: a(AA) b(AB1) b(AB2)
the children of b in two_nodes_multi_edges_graph are: a(BA) b(BB)